src/enumeration.jl

```diff
@@ -51,7 +64,7 @@ all_subtypes(ts::Vector, scfg :: SearchCfg, result :: Channel) = begin
             # Skip and push a marker describing the case to the caller.
             #
             #   - skip unionalls due to search config
-            unionalls = filter(t -> typeof(t) == UnionAll, tv)
+            unionalls = filter(t -> t isa UnionAll, tv)
```

Comment:
I wonder if there's any difference in practice, but your version is certainly preferable as more succinct.